jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  moduleNameMapper: {
+    '@core/(.*)': '<rootDir>/src/app/core/$1',
+  },
+  preset: 'jest-preset-angular',
+  setupFilesAfterEnv: ['<rootDir>/setup-jest.ts'],
```

Comment:
will use need to replace rootDir here?

Reply:
The <rootDir> property is just the base directory. This should only need to be changed if the user starts working with project modules, which is a more complicated setup anyway.

Reply:
Cool.. merging PR. What command do you use to test it? I will add it in build workflow?